_includes/payment-order-purchase.md

```diff
@@ -19,8 +19,8 @@ Content-Type: application/json
         "vatAmount": 375,
         "description": "Test Purchase",
         "userAgent": "Mozilla/5.0...",
-        "language": "sv-SE",{% if documentation_section == "payment-menu" %}
-        "instrument": "CreditCard"{% endif %}
+        "language": "sv-SE",
+        "instrument": null,
```

Comment:
I believe this `if` statement is still required, or else `instrument` will be enabled for Checkout as well?

```suggestion
        "language": "sv-SE",{% if documentation_section == "payment-menu" %}
        "instrument": null{% endif %}
```